### PR TITLE
Fix visual mode activated on insert mode

### PR DESCRIPTION
**Pull request by frarees**

### CODE REVIEW

File: src/mode/modeVisual.ts [hunk 1]
```diff
@@ -40,6 +40,9 @@ export class VisualMode extends Mode {
     }
 
     shouldBeActivated(key: string, currentMode: ModeName): boolean {
+        if (currentMode !== ModeName.Normal) {
+            return false;
+        }
         return key === "v";
```
Comment (jpoon) [inline src/mode/modeVisual.ts]: I think it's simpler if you do this:

`return key === "v" && currentMode === ModeName.Normal`

Can you also add in the `modeNormal.ts` that is you are in Visual Mode and `v` is entered, it should go to Normal Mode?

Comment (johnfn) [inline src/mode/modeVisual.ts]: Whoa, been using vim for ages and never knew that `v` in visual mode brings you out of visual mode!

**Comment by frarees** [comment]
@jpoon yeah let me revise it. Later today I will push some changes.

File: src/mode/modeVisual.ts (hunk 1 above)
Comment (jpoon) [inline src/mode/modeVisual.ts]: I'm in the same boat -- learning a lot of little things here and there building out this extension.

**Comment by jpoon** [comment]
Thanks @frarees

**Pull request merged**